src/app/pages/chips-page/chips-page.component.html

```diff
@@ -1 +1,55 @@
-<h1>Chips</h1>
+<h1>Start testing Chips by exploring the actions below</h1>
+<p>
+  Action chips are interactive UI elements triggering actions tied to main content. Compact
+  and context-specific, they enhance user experience by providing additional functionality.
+  For instance, in a music app, a chip may allow users to add songs to a playlist, or share
+  them, thus seamlessly integrating extra features.
+</p>
+<div class="tds-u-flex tds-u-flex-gap-1">
+  <tds-chip size="lg" onclick="alert('Clicked chip!')">
+    <tds-icon slot="prefix" name="lock_inactive" size="16px"></tds-icon>
+    <span slot="label">Lock</span>
+  </tds-chip>
+  <tds-chip size="lg" onclick="alert('Clicked chip!')">
```

Comment:
```suggestion
  <tds-chip size="lg" (click)="alert('Clicked chip!')">
```

Reply:
These kinds of thing I feel can just be commited straight away, no need to wait for me to get back to it.